src/templates/blog-post.js

```diff
@@ -1,26 +1,45 @@
 import React from 'react';
 import { graphql } from 'gatsby';
 import PropTypes from 'prop-types';
+import '../components/styles/bootstrap.css';
 import Layout from '../components/layout';
+import BlogFooter from '../components/Blog-Footer';
 
 export default function BlogPost(props) {
   const post = props.data.markdownRemark;
   return (
-    <Layout>
-      <div className="container" style={{ color: 'blanchedalmond', marginTop: '5vh' }}>
-        <h1>{post.frontmatter.title}</h1>
-        <div dangerouslySetInnerHTML={{ __html: post.html }} />
+    <Layout mainImgUrl={ post.frontmatter.imageUrl } mainImgAlt={ post.frontmatter.imageAlt }>
+      <div className='container-fluid'>
+        <div className='row'>
+          <div className='col-12'>
+            {post.frontmatter.title} ****THIS TITLE HAS NOT YET BEEN STYLED****
```

Comment:
Would probably remove this note, or keep it in a separate branch for now? I know the blog is still a WIP, but it's a good practice to keep notes to self and non-working features out of your main/master branch.